feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -50,55 +77,51 @@ fun OfficeView() {
  * @param onEmailButtonClicked launches the Email intent
  * @author Yaritza Moreno
  * Modified by:
```

Comment:
Remove line 79 **Modified by:** already exists **@modififiedBy** below next line

Reply:
got it